serialization.md

```diff
@@ -37,20 +37,16 @@ Address/Hash Helpers
     rule [#newAddr]:        #newAddr(ACCT, NONCE) => #addr(#parseHexWord(Keccak256(#rlpEncode([#addrBytes(ACCT), NONCE]))))
     rule [#newAddrCreate2]: #newAddr(ACCT, SALT, INITCODE) => #addr(#parseHexWord(Keccak256("\xff" +String #unparseByteStack(#addrBytes(ACCT)) +String #unparseByteStack(#wordBytes(SALT)) +String #unparseByteStack(#parseHexBytes(Keccak256(#unparseByteStack(INITCODE)))))))
 
-    syntax Account ::= #sender ( Int , Int , Int , Int , Account , Int , String , Int , JSONs , Int , ByteArray, ByteArray )    [function]
-                     | #sender ( String , Int , String , String )                                                               [function, klabel(#senderAux) ]
-                     | #sender ( String )                                                                                       [function, klabel(#senderAux2)]
- // -----------------------------------------------------------------------------------------------------------------------------------------------------------
-    rule #sender(_, _, _, _, _, _, _, _, _, TW => TW +Int 27, _, _)
+    syntax Account ::= #sender ( TxData , Int , ByteArray , ByteArray ) [function, klabel(#senderTxData)]
```

Comment:
This type of pattern is the reason for the new transaction symbols. While the new symbols may be just as verbose if not more, hopefully they're more clear whereas what was happening here is kind of opaque.